cmd/exec_test.go

```diff
@@ -0,0 +1,93 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/SAP/cloud-mta-build-tool/internal/archive"
+)
+
+var _ = Describe("cp command", func() {
+	BeforeEach(func() {
+		copyCmdSrc = getTestPath("mtahtml5")
+		copyCmdTrg = getTestPath("result")
+		copyCmdPatterns = []string{}
```

Comment:
It's here just in case you add a test and forget to initialize it, so it won't contain the value from the previous test (since it's a global variable) and the test will accidentally pass